qutepart/syntax/cParser.c

```diff
@@ -2756,26 +2756,27 @@ Context_appendSegment(PyObject* segmentList, int count, PyObject* format)
 }
 
 static void
-Context_appendTextType(int fromIndex, int count, PyObject* textTypeMap, char textType)
+Context_appendTextType(Py_ssize_t fromIndex, Py_ssize_t count, PyObject* textTypeMap, Py_UNICODE textType)
 {
-    int i;
+    Py_ssize_t i;
+    return;
```

Comment:
Yep, bad stuff. It's probably best to throw this away, or at least do it slowly, and fix the compiler complaints at a later time when I'm not in too much of a hurry.